atsign-foundation/at_app_cli
# Added at_onboarding_flutter and at_contacts_flutter sample apps

templates:

- [x] at_contacts_flutter
- [x] at_onboarding_flutter
- [x] chit_chat
- [x] snackbar
- [x] snackbar_sender

**- What I did**

Awaiting closure of #70 first.

- Added two samples from at_widgets repository
  - at_onboarding_flutter
  - at_contacts_flutter
- Added json-schemas for brick.yaml and template.yaml to make template maintenance easier
- Cleaned up the build configurations for all templates

**- How I did it**

- Copied the lib folder from each example, wrote the necessary config files for each template
- Built and bundled at_app

**- How to verify it**

1. Pull this branch
2. List the samples with `dart run at_app/bin/at_app.dart create --list-samples`
3. Run `dart run at_app/bin/at_app.dart create -s <sample> <path/to/output>` for each sample
4. Run each generated sample app and verify that they work

**- Description for the changelog**
Added at_onboarding_flutter and at_contacts_flutter sample apps
